lib/language_server/linter/ruby_wc.rb

```diff
@@ -27,7 +28,7 @@ def initialize(source)
 
       def call
         error_message.scan(/.+:(\d+):\s*(.+?)[,:]\s(.+)/).map do |line_num, type,  message|
-          Error.new(line_num: line_num.to_i - 1, message: message, type: type)
+          Error.new(line_num: line_num.to_i - 1, character: get_character_from_message(error_message), message: message, type: type)
```

Comment:
Can you implement `start` and `end` instead of `character`?

Reply:
Yes.
Implementing this is easy.
But I don't know what to do with the value of end.
Is the end value acceptable at the end of the line?

Reply:
If ruby tells us the position, `start + 1` is `end`.
If doesn't, `start` is 0 and `end` is the number of characters which the line has.